napari/layers/image/experimental/octree_image.py

```diff
@@ -361,9 +361,10 @@ def _update_draw(
 
         """
         # Compute our 2D corners from the incoming n-d corner_pixels
+        displayed_sorted = sorted(self._slice_input.displayed)
```

Comment:
This used the sorted displayed dimensions, whereas the equivalent code in `Layer` uses the unsorted ones. I'm not familiar enough with the experimental async implementation to know if that's intended or not. But there should be not functional change from main here.